src/app/dev/index.js

```diff
@@ -43,7 +43,10 @@ if (setCurrentProject.before) {
 if (setCurrentProject.after) {
   ProjectsRegistry.onafter('setCurrentProject', (project) => setCurrentProject.after(project));
 }
-
-GUI.once('ready', () => { document.body.classList.replace('skin-yellow', 'skin-blue'); });
+/**
+ * Comment because we leave g3w-admin to setup the skin color of current user type (es: admin, etc...)
+ * In this way also development environment has the same skin color of production
+ */
+//GUI.once('ready', () => { document.body.classList.replace('skin-yellow', 'skin-blue'); });
```

Comment:
Wasn't that the same behavior prior to this pull request?

https://github.com/g3w-suite/g3w-client/blob/58887a828986cf9c055a3d189da062fdf56a0ba2/src/index.dev.html#L22

https://github.com/g3w-suite/g3w-client/blob/58887a828986cf9c055a3d189da062fdf56a0ba2/src/index.prod.html#L34

That's why otherwise it becomes difficult to quickly distinguish the development environment from the production one?

```
# EXAMPLE 1:
# project_group = "countries";
# project_type  = "qdjango";
# project_id    = "1"

http://localhost:8000/en/map/countries/qdjango/1 # g3w-admin  (production)
http://localhost:3000/en/map/countries/qdjango/1 # g3w-client (development)
```

Reply:
@Raruto you are right, but the previous behavior before pull request is to set class skin-blue color hardcoded on index.dev.html template. 
Comment _GUI.once('ready', () => { document.body.classList.replace('skin-yellow', 'skin-blue'); });_ it useful to:

1. Have the same skin color of the application based on the user logged (or anonymous). Prior the pull request was not possible due a two template index.dev.html for devolopment and index.prod.html for production.
2. Solve the problem of user logged. In case of user logged that is not administrator (skin-yellow) but another type of user  (ex. viewer, etc ..), the skin color of applicvation can be skin-red, skin-purple, skin-green, so the **replace('skin-yellow', 'skin-blue')** doesn't work

Reply:
Ok, anyway on this point we can see if there is room for improvement in a subsequent pull request

Reply:
@Raruto ok